math/src/elliptic_curve/edwards/curves/bandersnatch/curve.rs

```diff
@@ -0,0 +1,129 @@
+pub use super::field_extension::FqField;
+use crate::elliptic_curve::edwards::point::EdwardsProjectivePoint;
+use crate::elliptic_curve::traits::IsEllipticCurve;
+use crate::{elliptic_curve::edwards::traits::IsEdwards, field::element::FieldElement};
+
+pub type BaseBandersnatchFieldElement = FqField;
+
+#[derive(Clone, Debug)]
+pub struct BandersnatchCurve;
+
+impl IsEllipticCurve for BandersnatchCurve {
+    type BaseField = BaseBandersnatchFieldElement;
+    type PointRepresentation = EdwardsProjectivePoint<Self>;
+
+    // Values are from https://github.com/arkworks-rs/curves/blob/5a41d7f27a703a7ea9c48512a4148443ec6c747e/ed_on_bls12_381_bandersnatch/src/curves/mod.rs#L120
+    // Converted to Hex
+    fn generator() -> Self::PointRepresentation {
+        Self::PointRepresentation::new([
+            FieldElement::<Self::BaseField>::new_base(
+                "29C132CC2C0B34C5743711777BBE42F32B79C022AD998465E1E71866A252AE18",
+            ),
+            FieldElement::<Self::BaseField>::new_base(
+                "2A6C669EDA123E0F157D8B50BADCD586358CAD81EEE464605E3167B6CC974166",
+            ),
+            FieldElement::one(),
+        ])
+    }
```

Comment:
The generators don't seem to match. This is in projective coordinates, and the one linked are in affine

Reply:
Alright, the conversion from affine to projective is good